src/main/java/br/edu/utfpr/autorepairshop/model/Employees.java

```diff
@@ -0,0 +1,52 @@
+package br.edu.utfpr.autorepairshop.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "employees")
+@Getter
+@Setter
+@Data
+@NoArgsConstructor
+public class Employees {
+	@Id
+	@Column(name = "id_employee")
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long idEmployee;
+
+	@JoinColumn(name = "id_credentials")
+	@OneToOne
+	private Credentials idCredentials;
+
+	@Column(name = "id_workshop")
+	private Long idWorkshop;
```

Comment:
Não entendi o que representa. Se for oficina, deve ser representado por um relacionamento @ManyToOne, não como Long.